src/pages/welcome.astro

```diff
@@ -0,0 +1,439 @@
+---
+import i18next, { t, changeLanguage } from 'i18next'
+import { HeadHrefLangs } from 'astro-i18next/components'
+import '/node_modules/@interledger/docs-design-system/src/styles/teal-theme.css'
+import '../styles/webm.css'
+
+interface Props {
+  title: string
+  description: string
+  image?: string
+}
+
+const canonicalURL = new URL(Astro.url.pathname, Astro.site)
+const {
+  title = t('site.title'),
+  description = t('site.description'),
+  image = '/img/wm-social.png',
+} = Astro.props
+changeLanguage('en')
+---
+
+<html lang={i18next.language} data-theme='light'>
+  <meta charset='utf-8' />
+  <meta name='viewport' content='width=device-width,initial-scale=1' />
+  <link rel='icon' type='image/svg+xml' href='/favicon.svg' />
+  <meta name='generator' content={Astro.generator} />
+
+  <link rel='canonical' href={canonicalURL} />
+
+  <title>{title}</title>
+  <meta name='title' content={title} />
+  <meta name='description' content={description} />
+
+  <meta property='og:type' content='website' />
+  <meta property='og:url' content={Astro.url} />
+  <meta property='og:title' content={title} />
+  <meta property='og:description' content={description} />
+  <meta property='og:image' content={new URL(image, Astro.url)} />
+
+  <meta property='twitter:card' content='summary_large_image' />
+  <meta property='twitter:url' content={Astro.url} />
+  <meta property='twitter:title' content={title} />
+  <meta property='twitter:description' content={description} />
+  <meta property='twitter:image' content={new URL(image, Astro.url)} />
+
+  <HeadHrefLangs />
+
+  <body>
+    <header>
+      <nav class='site-nav'>
+        <a href='/' class='site-title'>
+          <img src='/img/wm-logo.svg' alt='Web Monetization logo' />
+          <span>Web Monetization</span>
+        </a>
+        <button
+          id='menuToggle'
+          class='menu-toggle'
+          aria-controls='nav-menu'
+          aria-expanded='true'
+          aria-label='Toggle Menu'
+          title='Toggle Menu'
+        >
+          <div id='menuIcon' class='menu-icon'>
+            <span></span>
+            <span></span>
+            <span></span>
+            <span></span>
+          </div>
+        </button>
+        <ul id='navMenu' class='nav-menu collapsed'>
+          <li class='nav-link__docs'><a href='/docs/'>Documentation</a></li>
+          <li class='nav-link__docs'>
+            <a href='https://community.interledger.org/t/webmonetization'>
+              Community
+            </a>
+          </li>
+          <li class='nav-link__github'>
+            <a href='https://github.com/WICG/webmonetization/'>
+              <img
+                src='/img/icon-github.svg'
+                alt='Link to Github repository'
+                />
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class='hero'>
+        <div class='heroContent'>
+          <img
+            alt='Loading'
```

Comment:
Consider adding width/height to prevent layout shift on image load. If we can use same size for all, that'll be even better.